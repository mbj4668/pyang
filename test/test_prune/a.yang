module a {
  namespace urn:a;
  prefix a;

  include atop;
  include asub;

  feature foo;
  feature foz;

  grouping gg {
    leaf bar-gggg {
      type string;
    }
  }

  container x {
    leaf bar-leaf {
      if-feature bar;
      type string;
    }
    uses gg {
      if-feature bar;
    }
    leaf baz {
      if-feature foo;
      type string;
    }
    leaf bubba {
      type string;
    }
  }

  augment /x {
    if-feature bar;
    container bar-y;
  }

  rpc bar-rpc {
    if-feature bar;
  }
  rpc foo-rpc {
    if-feature foo;
  }
}
