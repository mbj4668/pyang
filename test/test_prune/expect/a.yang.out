module: a
  +--rw top
  +--rw x
     +--rw bubba?   int32

  rpcs:
    +---x foo-rpc {foo}?

  notifications:
    +---n fox-notif {fox}?
